<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgFullscreenSettings</class>
 <widget class="QWidget" name="WdgFullscreenSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select which parts of Krita will be hidden in canvas-only mode.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkMenu">
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkToolbar">
     <property name="text">
      <string>Toolbars</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkDockers">
     <property name="text">
      <string>Toolbox and dockers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkScrollbars">
     <property name="text">
      <string>Scrollbars</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkTitlebar">
     <property name="text">
      <string>Titlebar (hiding the titlebar will make Krita go full-screen)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkStatusbar">
     <property name="text">
      <string>Statusbar</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>85</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
